infra/gcp/terraform/k8s-infra-oci-proxy-prod/terraform.tfvars

```diff
@@ -162,8 +235,14 @@ cloud_run_config = {
       }
     ]
   }
+  // GCP southamerica-west1 is Santiago, Chile
   southamerica-west1 = {
     environment_variables = [
+      {
+        name = "DEFAULT_AWS_BASE_URL",
+        // AWS us-east-1 is Virginia, USA
```

Comment:
This one is pretty surprising and maybe comment worthy

Reply:
Fair, we don't have a bucket in south america currently https://github.com/kubernetes/registry.k8s.io/blob/8d1d604a30942e89b7e085534e4d39f2819440e4/cmd/archeio/app/buckets.go#L38-L67

I don't think any of the other regions we have are closer

Reply:
Regions we have a bucket in were based on traffic data xref https://github.com/kubernetes/registry.k8s.io/issues/39

Reply:
Added a self-referential comment to this discussion with context links

Reply:
@BenTheElder there's one in sao paolo `sa-east-1`

Reply:
do we mean we don't have a bucket there and choose to serve from us-east-1?

Reply:
Yes, to clarify there is an AWS region in South America but kubernetes does not have a bucket there, while we did wind up having cloud run / AR there.

Most places we have reasonably close proximity between AR/Cloud Run and S3, but not here.

Reply:
(Also we only have a few QPS here, and previously k8s.gcr.io only had EU,US,Asia multi-regional storage, so while this feels off it's probably not the most pressing)

Reply:
this is now https://github.com/kubernetes/registry.k8s.io/issues/172